src/attributes.cpp

```diff
@@ -2413,11 +2434,14 @@ namespace attributes {
                 // determine the function name
                 std::string name = attribute.exportedName();
 
+                // determine if return invisible
+                bool invisible = function.type().isVoid() || attribute.invisible();
```

Comment:
Maybe call this `invisibleOrVoid` which would make it clearer below that what you add "just" generalizes the `void` case?

Reply:
Or, looking at other boolean toggles in the files:  `isInvisibleOrVoid` ?

Reply:
` isInvisibleOrVoid` looks good.